README.md

```diff
@@ -12,10 +12,25 @@
   - In project folder run `pipenv install`
 
 - Copy .env.example into new .env file
+- It's highly recommended you have docker installed it makes environment setup much easier: https://www.docker.com/get-started
+- Install Neo4j
+  - Neo4J is our GraphDB you can find more info on it here: https://neo4j.com/
+  - To simply run the application locally setup a docker container with neo4j with this command: 
+    ```
+    docker run \
```

Comment:
Is this command Windows friendly?

Reply:
Oh good question, how do you setup docker containers on windows? It's probably not, just copy pasted from the neo4j website